.goreleaser.yaml

```diff
@@ -0,0 +1,81 @@
+project_name: apm-lambda-extension
```

Comment:
**issue**: what happened to reproducible builds ? AFAICS this also does not support `SOURCE_DATE_EPOCH`. Two identical commits should produce the same build output regardless of time or host the build was started on.
Be it a docker image or a zip archive.

Reply:
Thank you. This go lost in translation. I will try to replicate this behaviour.

Reply:
For my understanding:

to make the build reproducible, I have to touch the artifacts with the timestamp of the git log, right?
Is there anything else I am missing?

Reply:
This is my attempt, let me know if this works!

https://github.com/elastic/apm-aws-lambda/pull/362/commits/1c33f8f812663035211c1cac4251c3c284da3637

cc @v1v